incubator/selenium/templates/NOTES.txt

```diff
@@ -0,0 +1,89 @@
+Selenium Hub can be accessed via port {{ .Values.hub.port }} on the following 
+DNS name from within your cluster:
+
+- http://{{ template "hub.fullname" . }}.{{ .Release.Namespace }}:{{ .Values.hub.port }}
+
+You can easily connect to the remote instance from your browser. Forward the
+webserver port to localhost:4444
+
+- kubectl port-forward --namespace {{ .Release.Namespace }} \
+  $(kubectl get pods --namespace {{ .Release.Namespace }} \
```

Comment:
Minor nit but helpful to users: remove any special formatting in the NOTES examples so folks can copy/paste. e.g. so this:

```
You can easily connect to the remote instance from your browser. Forward the
webserver port to localhost:4444

- kubectl port-forward --namespace default \
```

becomes

```
You can easily connect to the remote instance from your browser. Forward the
webserver port to localhost:4444

kubectl port-forward --namespace default \
```